apps/pom.xml

```diff
@@ -600,6 +605,11 @@ checkJavaFormat
 						<argLine>${maven-test.jvm-args.env-specific}</argLine>
 						<trimStackTrace>false</trimStackTrace>
 						<redirectTestOutputToFile>true</redirectTestOutputToFile>
+						<!-- run ITs w/multiple threads -->
+						<useUnlimitedThreads>true</useUnlimitedThreads>
+						<parallel>all</parallel>
+						<threadCountClasses>8</threadCountClasses>
+						<threadCountMethods>10</threadCountMethods>
```

Comment:
This will likely break the server-war IT tests, which are not safe to run with multiple threads. We also probably don't want to run the server-war tests with threads anyhow; we want to ensure just one server and DB gets spun up for the server-war tests for speed; the speed of spinning up a server per class would almost certainly be slower than the speed gained by threading.

Reply:
sounds good; when your PR is merged, I'll rebase and fix the POMs in bfd-server to mimic what RDA does.

Reply:
It's merged as of a few days ago.